bower.json

```diff
@@ -1,16 +1,43 @@
 {
   "name": "paper-dropdown-menu",
-  "private": false,
+  "version": "1.0.0",
+  "description": "",
+  "authors": [
+    "The Polymer Authors"
+  ],
+  "keywords": [
+    "web-components",
+    "web-component",
+    "polymer"
+  ],
+  "main": "paper-dropdown-menu.html",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/PolymerElements/paper-dropdown-menu"
+  },
+  "license": "http://polymer.github.io/LICENSE.txt",
+  "homepage": "https://github.com/PolymerElements/paper-dropdown-menu",
   "dependencies": {
-    "polymer": "Polymer/polymer#master",
-    "core-a11y-keys": "Polymer/core-a11y-keys#master",
-    "core-collapse": "Polymer/core-collapse#master",
-    "core-focusable": "Polymer/core-focusable#master",
-    "core-icon": "Polymer/core-icon#master",
-    "core-icons": "Polymer/core-icons#master",
-    "core-menu": "Polymer/core-menu#master",
-    "paper-dropdown": "Polymer/paper-dropdown#master",
-    "paper-item": "Polymer/paper-item#master",
-    "paper-shadow": "Polymer/paper-shadow#master"
+    "polymer": "polymer/polymer#^1.0.0",
+    "paper-menu-button": "polymerelements/paper-menu-button#^1.0.0",
+    "iron-icons": "polymerelements/iron-icons#^1.0.0",
+    "iron-selector": "polymerelements/iron-selector#^1.0.0",
+    "paper-input": "polymerelements/paper-input#^1.0.9",
+    "iron-a11y-keys-behavior": "polymerelements/iron-a11y-keys-behavior#^1.0.0",
+    "iron-icon": "polymerelements/iron-icon#^1.0.0",
+    "paper-ripple": "polymerelements/paper-ripple#^1.0.0",
+    "paper-styles": "polymerelements/paper-styles#^1.0.0"
```

Comment:
Hehe, this needs to be cleaned up a little. There has been a lot of experimentation.